app/javascript/crayons/Buttons/__stories__/Buttons.mdx

```diff
@@ -14,35 +14,50 @@ That is: `<CTA />` or `<Link />` should be used where navigation is elicited, an
 For example, a 'Submit' control on 'Sign in' view (form) would suit a `<Button />` because its job is to submit the form. 
 However, 'Register' call-to-action—taking the user to a different form, in a different page or screen—would benefit from `<CTA />` or `<Link />` semantics.
 
-Do not use `<Button />s` as navigational elements. Instead, use Links (or CTAs) when the desired action is to take the user to a new page.
+Do not use `<Button />s` as navigational elements. Instead, use `<Link />s` (or `<CTA />s`) when the desired action is to take the user to a new page.
 
 ## Variants
 
+### Default
+
+The _Default_ variant is not very prominent, and therefore can be used for less prominent actions.
+The _Default_ buttons are visually light (greyish) and you can have as many of them as you want (within common sense :D). 
+They can (but don't have to) be used in conjunction with a _Primary_ button and less often with _Secondary_ buttons (use your common sense again or ping your designer).
+When combined with the _Primary_ or the _Secondary_, the _Default_ variant should be used for any remaining actions.
+
+Whenever used, make sure it has sufficient color contrast against the background it's used on.
+
 ### Primary
 
-Primary button has special look: high contrast (against rest of the view/component) and it's filled with accent color. 
+The _Primary_ variant has special look: high contrast (against rest of the view/component) and it's filled with accent color. 
 
-Ideally there should be only one Primary button per entire component, or sometimes even per entire view.
-Primary buttons should be used for the principal call to action on the page.
+Ideally there should be only one _Primary_ button per entire component, or sometimes even per entire view.
+Primary buttons should be used for the principal action on the page.
 
-### Default
+### Secondary
+
+The _Secondary_ variant is visually in between _Primary_ and _Default_ buttons. It's not as loud as _Primary_ button but definitely stands out more than _Default_ button.
```

Comment:
**_Quick suggestion_**: let's add "the" before the button names here for consistency and readability. 😊 
```suggestion
The _Secondary_ variant is visually in between the _Primary_ and the _Default_ buttons. It's not as loud as the _Primary_ button but definitely stands out more than the _Default_ button.
```